README.md

```diff
@@ -1,2 +1,66 @@
 # business-dashboard-ui
 Contains business filing ledger, addresses, people, documents, and launch points for new filings
+
+# Nuxt 3 Minimal Starter
+
+Look at the [Nuxt 3 documentation](https://nuxt.com/docs/getting-started/introduction) to learn more.
+
+## Setup
+
+Make sure to install the dependencies:
+
+```bash
+# pnpm
+pnpm install
+```
+
+## Development Server
+
+Start the development server on `http://localhost:3000`:
+
+```bash
+# pnpm
+pnpm run dev
+```
+
+## Production
+
+Build the application for production:
+
+```bash
+# pnpm
+pnpm run build
+```
+
+Locally preview production build:
+
+```bash
+# pnpm
+pnpm run preview
+```
+
+Check out the [deployment documentation](https://nuxt.com/docs/getting-started/deployment) for more information.
+
+# Proposed Technical Plan for Entity UIs
```

Comment:
Maybe update this to say that this is specific UI rather then plan of entire entity stuff ?

Reply:
Good idea will update

Reply:
removed this entirely as its not really the right place for it. Tracking this with the entity team in this ticket: https://github.com/bcgov/entity/issues/21469